Tasks/KubernetesManifestV1/_buildConfigs/Node20/package-lock.json

```diff
@@ -302,22 +291,6 @@
         "uuid": "^3.0.1"
       }
     },
-    "node_modules/azure-pipelines-tasks-azure-arm-rest/node_modules/minimatch": {
```

Comment:
So there were three entries for minimatch before?

Reply:
These were auto generated.